Prox/Prox/RemoteConfigKeys.swift

```diff
@@ -42,6 +42,9 @@ class RemoteConfigKeys {
     // This is measured in minutes. The default value is 1 minute.
     public static let youAreHereWalkingTimeMins = RemoteConfigInt(key: "you_are_here_walking_time_mins", defaultValue: 1)
 
+    // The distance the user must travel before the travel times on a location are reloaded.
+    public static let travelTimeExpirationDistance = RemoteConfigDouble(key: "travel_time_expiration_distance", defaultValue: 0.2) // default: ~2.5min walk.
```

Comment:
can we have the unit of measurement in the key name here, like with the other keys? this is so it makes it easy to update without having to look at the code.  So, `travel_time_expiration_distance_meters` or whatever the unit is.